packages/web/hooks/queries/assets/use-coin-price.ts

```diff
@@ -9,8 +9,8 @@ export function useCoinPrice(coin?: CoinPretty) {
     },
     {
       enabled: Boolean(coin?.currency),
-      cacheTime: 1000 * 5, // 5 seconds
-      staleTime: 1000 * 5, // 5 seconds
+      cacheTime: 1000 * 3, // 3 second
+      staleTime: 1000 * 3, // 3 second
```

Comment:
Please not that this only lowers the frequency at the FE fetches the prices, this won't affect the cached value in our tRPC function. Is this expected?

Reply:
Yes, let me explain.

For displaying prices in the swap tool we used to have 2 pricing sources:
1. This
2. tRPC 


 [this PR](https://github.com/osmosis-labs/osmosis-frontend/pull/3070) removes the swap tool fiat pricing display dependency from tRPC.

Now, we only use 1). This PR simply makes it update more frequently